runtime/doc/autocmd.txt

```diff
@@ -553,6 +554,9 @@ CursorMoved			After the cursor was moved in Normal or Visual
 CursorMovedI			After the cursor was moved in Insert mode.
 				Not triggered when the popup menu is visible.
 				Otherwise the same as CursorMoved.
+							*CursorMovedT*
+CursorMovedT			After the cursor was moved in Terminal mode.
+				Equivalent to CursorMovedI.
```

Comment:
Yeah, I mostly mentioned it because of the documentation. I'm playing with it, and I can't make a completion popup to appear at all, which is to be expected since it seems like the `:terminal` command handles all keys and passes them down to the running process.